external/netty/test.properties

```diff
@@ -4,7 +4,4 @@ test_commands="mvn clean package"
 home_path="\${WORKDIR}"
 test_results="testResults"
 tag_version="4.1"
-debian_packages="git maven autoconf automake libtool make tar gcc-multilib libaio-dev"
-alpine_packages="bash git maven"
-centos_packages="git maven"
-ubi_packages="git maven"
+ubuntu_packages="git maven"
```

Comment:
same here, the reason change  `git maven autoconf automake libtool make tar gcc-multilib libaio-dev` to `git maven`?